frontend/src/index.css

```diff
@@ -1,23 +1,23 @@
 body {
-    margin: 0;
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-      sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-  }
-  
-  code {
-    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-      monospace;
-  }
-  .color-nav{
-    background-color: rgb(0, 54, 96);
-  } 
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
 
-  .btn-primary, .btn-primary:hover, .btn-primary:focus{
-    background-color: #34859b;
-    border-color: #34859b;
-  } 
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+.color-nav {
+  background-color: rgb(0, 54, 96);
+}
 
-  
+.btn-primary,
+.btn-primary:hover,
+.btn-primary:focus {
+  background-color: #34859b;
+  border-color: #34859b;
+}
```

Comment:
I see that this is only formatting differences.  But this kind of stuff clutters up a PR and makes it harder to review.

In industry, this is considered a sloppy practice, so I'd encourage you to avoid it.